XCUITests/NewTabSettings.swift

```diff
@@ -29,6 +29,19 @@ class NewTabSettingsTest: BaseTestCase {
         waitForNoExistence(app.collectionViews.cells["TopSitesCell"].collectionViews.cells["youtube"])
         waitForNoExistence(app.staticTexts["Highlights"])
     }
+    
+    func testChangeNewTabSettingsShowFirefoxHome() {
+        navigator.goto(NewTabSettings)
+        navigator.performAction(Action.SelectNewTabAsBlankPage)
```

Comment:
And also same comment as before, if we should check that the blank page actually open before selecting FF as new tab...that first part may be already covered in other tests...so as you prefer and the way the test is cleaner and more stable...